code/modules/food_and_drinks/machinery/processor.dm

```diff
@@ -50,6 +53,13 @@
 	if(in_range(user, src) || isobserver(user))
 		. += span_notice("The status display reads: Outputting <b>[rating_amount]</b> item(s) at <b>[rating_speed*100]%</b> speed.")
 
+/obj/machinery/processor/Exited(atom/movable/gone, direction)
+	LAZYREMOVE(processor_contents, gone)
+
+/*
+	Spawns the items based on the passed processor recipes, and transfers materials and reagents
+	If the input was a mob, gibs them, otherwise, deletes the item
+*/
```

Comment:
```suggestion
/*
*	Spawns the items based on the passed processor recipes, and transfers materials and reagents
*	If the input was a mob, gibs them, otherwise, deletes the item
*/
```

lack of asterisks gives me the heebies even though it's fine technically. sorry for the nit here